relay-quotas/src/is_rate_limited.lua

```diff
@@ -28,24 +28,36 @@
 -- The result is a Lua table/array (Redis multi bulk reply) that specifies
 -- whether or not the item was *rejected* based on the provided limit.
 assert(#KEYS % 2 == 0, "there must be 2 keys per quota")
-assert(#ARGV % 3 == 0, "there must be 3 args per quota")
-assert(#KEYS / 2 == #ARGV / 3, "incorrect number of keys and arguments provided")
+assert(#ARGV % 4 == 0, "there must be 4 args per quota")
+assert(#KEYS / 2 == #ARGV / 4, "incorrect number of keys and arguments provided")
+
+-- parase the incoming string into boolean, returns `nil` if could not parse which is also considered `false`
```

Comment:
```suggestion
-- parse the incoming string into boolean, returns `nil` if could not parse which is also considered `false`
```

Reply:
done in [6875902](https://github.com/getsentry/relay/pull/1521/commits/6875902fb62f52263758c3711f8e898777003bcc)